rust/navigator/src/lib.rs

```diff
@@ -61,3 +64,16 @@ pub fn update_seed_names(seed_names: Vec<String>) -> Result<()> {
 
     Ok(())
 }
+
+/// Export key info with derivations.
+pub fn export_key_info(
+    dbname: &str,
+    selected_names: Option<Vec<String>>,
+) -> Result<MKeysInfoExport> {
+    let export_all_addrs = export_all_addrs(dbname, selected_names)?;
+
+    let data = [&[0x53, 0xff, 0xde], export_all_addrs.encode().as_slice()].concat();
```

Comment:
@pgolovkin the data in the QR would now have this prefix, it is not yet described in the docs. It is needed for compatibility with UOS protocol so two Signer devices can talk to each other.